docs/usage/administration/useful_sql_for_admins.md

```diff
@@ -1,47 +1,78 @@
 ## Some useful SQL queries for Synapse Admins
```

Comment:
This is a bit scary that we document these SQL statements instead of adding APIs! Can any of these be removed and use the admin API instead?

I wonder if this should at least point to the admin API...

Reply:
For what it's worth, I think all the queries here are `SELECT`s. Though as you say elsewhere, an expensive SELECT can still cause pain!